content/Software Support/Upload/Errors-when-uploading-a-sketch.md

```diff
@@ -19,28 +19,32 @@ These checks will resolve the most common errors:
 
 If you still get an error, you'll find instructions for specific error messages below.
 
+---
+
 <a id="errors"></a>
 
-## Errors
+## List of errors
 
 > **Note:** _Exit status 1_ only indicates that the process has failed. Look for preceding lines in the error output for more specific information.
 
 <table>
 <tr>
 <td style="vertical-align: top;">
-<h3>Common</h3>
+<h3>Upload</h3>
 <ul>
-<li><a href="https://support.arduino.cc/hc/en-us/articles/360016955139-Error-exit-status-1">Error: exit status 1</a></li>
-<li><a href="https://support.arduino.cc/hc/en-us/articles/360020843439-Error-2-UNKNOWN-no-FQBN-provided">Error: 2 UNKNOWN: no FQBN provided</a></li>
-<li><a href="#no-device-found-on-port">No device found on &lt;port&gt;<br>Upload error: Failed uploading: uploading error: exit status 1</a></li>
+<li><a href="https://support.arduino.cc/hc/en-us/articles/360020831120-Failed-uploading-no-upload-port-provided">Failed uploading: no upload port provided</a></li>
+<li><a href="https://support.arduino.cc/hc/en-us/articles/6554944868252-Failed-uploading-uploading-error-exit-status-1">Upload error: Failed uploading: uploading error: exit status 1</a></li>
```

Comment:
```suggestion
<li><a href="https://support.arduino.cc/hc/en-us/articles/6554944868252-Failed-uploading-uploading-error-exit-status-1">Failed uploading: uploading error: exit status 1</a></li>
```
Maybe "Upload error" should be removed to match the error message

Reply:
Nice catch!